src/assets/locales/de/main.json

```diff
@@ -104,6 +104,22 @@
     "errorBar": "Something went wrong. Try again"
   },
   "content": {
+    "registration": {
+      "title": "Welcome to Catena-X",
+      "mainDescription": "Thank you for registering with us. You have started your registration process but a few things are still missing.",
+      "completeBtn": "Complete your registration",
+      "description": "At the current state of the registration you cannot access the portal. After completing all phases of the registration you will get limited access to the portal while your registration is being reviewed.",
+      "steps": {
+        "step1": "Verify your company data",
+        "step2": "Responsibilities & admin account",
+        "step3": "Role of your company",
+        "step4": "Upload IDs and certificates",
+        "step5": "Verify your entries",
+        "tooltipText": "Continue where you left"
+      },
+      "helpText": "If you have any questions regarding your registration please contact our Helpdesk:",
+      "email": "help@catena-x.com"
+    },
```

Comment:
@nidhigarg-bmw please transfer the messages to german

Reply:
@jjeroch Made the translations